src/interface/blas2/trsv.cpp.in

```diff
@@ -0,0 +1,42 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename trsv.cpp.in
+ *
+ **************************************************************************/
+#include "container/sycl_iterator.hpp"
+#include "sb_handle/sycl_blas_handle.hpp"
+#include "sb_handle/kernel_constructor.hpp"
+#include "interface/blas2_interface.hpp"
```

Comment:
I think that only `blas2_interface.hpp` will be required here and we can remove the rest. Could you please check if the code works after removing the unnecessary headers?

FYI: I know this is an issue with almost all of the operators in SYCL-BLAS, we can probably put up a PR later on to clean all of those files.

Reply:
[2626b52](https://github.com/codeplaysoftware/sycl-blas/pull/381/commits/2626b524238a7b5905c258206217bd6114b977c2) removes the unnecessary headers. `sb_handle/sycl_blas_handle.hpp` and `sb_handle/kernel_constructor.hpp` need to be present.